### PR TITLE
Update CI badges with Metal3

### PR DESCRIPTION
**Pull request by kashifest**
Airship name from CI jenkins view have been deprecated.

### CODE REVIEW

File: scripts/feature_tests/README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 ## Build Status
 
-[![Ubuntu V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_ubuntu/badge/icon?subject=Feature-tests)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_ubuntu/)
-[![Centos V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_centos/badge/icon?subject=Feature-tests-centos)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_centos/)
-[![Ubuntu V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_ubuntu/badge/icon?subject=Feature-tests-upgrade)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_ubuntu/)
+[![Ubuntu V1alpha4 build status](https://jenkins.nordix.org/view/Metal3/job/Metal3_main_feature_tests_ubuntu/badge/icon?subject=Feature-tests)](https://jenkins.nordix.org/view/Metal3/job/Metal3_main_feature_tests_ubuntu/)
```
Comment (furkatgofurov7) [inline scripts/feature_tests/README.md]: This is outdated, we use v1beta1 for feature tests

File: scripts/feature_tests/README.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
+[![Centos V1alpha4 build status](https://jenkins.nordix.org/view/Metal3/job/Metal3_main_feature_tests_centos/badge/icon?subject=Feature-tests-centos)](https://jenkins.nordix.org/view/Metal3/job/Metal3_main_feature_tests_centos/)
```
Comment (furkatgofurov7) [inline scripts/feature_tests/README.md]: same

File: scripts/feature_tests/README.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
...
+[![Ubuntu V1alpha4 build status](https://jenkins.nordix.org/view/Metal3/job/Metal3_main_feature_tests_upgrade_ubuntu/badge/icon?subject=Feature-tests-upgrade)](https://jenkins.nordix.org/view/Metal3/job/Metal3_main_feature_tests_upgrade_ubuntu/)
```
Comment (furkatgofurov7) [inline scripts/feature_tests/README.md]: same

File: scripts/feature_tests/README.md [hunk 4]
```diff
@@ -136,35 +136,35 @@ Recommended resource requirements for the host machine are: 8C CPUs, 32 GB RAM,
 
 We are running two main jobs for the **remediation/pivoting** framework testing.
```
Comment (furkatgofurov7) [inline scripts/feature_tests/README.md:137]: Would be nice also if you could update the test name mentions we run on feature tests in this whole document where we mention them, looks like they are missing some of them. We run:
- inspection ==> remediation ==> pivoting ==> node reuse ==> repivoting 

in feature tests

File: scripts/feature_tests/README.md (hunk 1 above)
Comment (kashifest) [reply scripts/feature_tests/README.md]: Actually this text was whole wrong. It should be main feature test.

Comment (kashifest) [reply scripts/feature_tests/README.md]: Done

File: scripts/feature_tests/README.md (hunk 2 above)
Comment (kashifest) [reply scripts/feature_tests/README.md]: Done

**Review by furkatgofurov7 — APPROVED** [review]
/approve

**Pull request merged**